transformer/index.js

```diff
@@ -1,123 +0,0 @@
-"use strict"
```

Comment:
These `transformer` utils no longer needed?

Reply:
We originally built our own, then https://github.com/ds300/react-native-typescript-transformer was built based on our work and made available for everyone's apps - we migrated to the module now

Reply:
very nice! I bet a lot of people were psyched on this one